components/symbl_ai/symbl_ai.app.mjs

```diff
@@ -25,6 +25,31 @@ export default {
         }));
       },
     },
+    memberId: {
+      type: "string",
+      label: "Member Id",
+      description: "The unique identifier of the member in the Conversation.",
+      async options({
+        page, conversationId,
+      }) {
+        const limit = 50;
+        const params = {
+          limit,
+          offset: limit * page,
+          order: "desc",
+        };
+        const { members } = await this.getMembers({
+          params,
+          conversationId,
+        });
+        return {
+          options: members.map((member) => ({
+            label: member.name,
+            value: member.id,
+          })),
+        };
```

Comment:
This can be simplified.
```suggestion
        return members.map((member) => ({
          label: member.name,
          value: member.id,
        })),
```

Reply:
Awesome. Thanks @michelle0927. Addressed in 9f0cf2a9.